Cargo.toml

```diff
@@ -22,7 +22,6 @@ edition = "2018"
 [dependencies]
 async-trait = "0.1.22"
 regex = "1"
-tokio = "0.1"
 futures = {version = "0.3", features = ["compat", "io-compat", "std"]}
 tokio02 = { package="tokio", version = "0.2.15", features = ["full"]}
```

Comment:
Please rename tokio02, tokio02util, tokio02tls to tokio, tokio-util, tokio-tls, and remove tokio-compat, tokio-util probably doesn't need the compat feature

Reply:
Done!